newton/examples/example_mujoco.py

```diff
@@ -365,6 +390,10 @@ def create_solver(
         nconmax = nconmax if nconmax is not None else ROBOT_CONFIGS[robot]["nconmax"]
         ls_parallel = ls_parallel if ls_parallel is not None else ROBOT_CONFIGS[robot]["ls_parallel"]
 
+        if env == "kitchen":
+            njmax += ROBOT_CONFIGS[env]["njmax"]
+            nconmax += ROBOT_CONFIGS[env]["nconmax"]
```

Comment:
Same here, better to just use 
```py
njmax += ROBOT_CONFIGS[env].get("njmax", 0)
nconmax += ROBOT_CONFIGS[env].get("nconmax", 0)
```
to not clutter the code here with env-specific logic.